Makefile

```diff
@@ -40,11 +40,11 @@ TEST_DIR = programs/tests
 # Default configuration of core, programs, and target. 
 # Override by modifying config.mk or pass varaible assignment as argument.
 # See file for description of variables.
-include config.mk
+include defconfig.mk
 
 # Construct core configuration string (used for directory naming).
 # Note: '?=' not used so string is only constructed once.
-CORE_CONFIG := $(THREADS)t$(if $(findstring true, $(FLEXPRET)),f)-$(ISPM_KBYTES)i-$(DSPM_KBYTES)d$(if $(findstring true, $(MUL)),-mul)-$(SUFFIX)
+CORE_CONFIG := $(THREADS)t$(if $(findstring true, $(FLEXPRET)),f)-$(ISPM_KBYTES)i-$(DSPM_KBYTES)d-$(STACK_SIZE)s$(if $(findstring true, $(MUL)),-mul)-$(SUFFIX)
```

Comment:
I changed this back to what it used to be and instead made a new file flexpret_swconfig.h in the flexpret lib. Also renamed flexpret_config.h -> flexpret_hwconfig.h. I couldn't simply place the SW config inside the previous flexpret_config.h since it is auto generated.

So now the configuration of the stack pointer is in the flexpret_swconfig.h instead of being passed to scala.

Reply:
Great!